kytos/cli/commands/napps/api.py

```diff
@@ -200,12 +200,14 @@ def list(cls, args):
         napps = [napp + ('[ie]',) for napp in mgr.get_enabled()]
         napps += [napp + ('[i-]',) for napp in mgr.get_disabled()]
 
-        # Sort, add description and reorder coloumns
+        # Sort, add description and reorder columns
         napps.sort()
         napps_ordered = []
         for user, name, status in napps:
-            napps_ordered.append((status, '{}/{}'.format(user, name),
-                                  mgr.get_description(user, name)))
+            description = mgr.get_description(user, name)
+            version = mgr.get_version(user, name)
```

Comment:
If _kytos.json_ does not have `version` key, I believe a KeyError will be raised (let's keep in mind we can be reading a local-created NApp). For several reasons (I want to be brief here), we don't need to catch this exception right now.

I agree we should check for an empty version, preferably with a unit test, and do something like:
```python
# Assuming version is a string.
napp_id = f'{user}/{name}-{version}' if version else f'{user}/{name}'
# If not enough columns:
napp_id = f'{user}/{name}'
if version:
    napp_id += f'-{version}'
```